antora.yml

```diff
@@ -0,0 +1,14 @@
+name: apps-manager
+title: Bonita Apps Manager
+version: 1.0
+asciidoc:
```

Comment:
This component is not publicly available, so it should be marked as a pre-release: https://docs.antora.org/antora/latest/component-prerelease/

Reply:
fixed